### PR TITLE
Add 'title' and 'text' to share_target params in manifest.json

### PR DESCRIPTION
**Pull request by cristicismas**
Should fix https://github.com/blenderskool/blaze/issues/74

### CODE REVIEW

File: client/src/manifest.json [hunk 1]
```diff
@@ -34,6 +34,8 @@
     "method": "POST",
     "enctype": "multipart/form-data",
     "params": {
+      "title": "Blaze",
```
Comment (abhishek-aa) [inline client/src/manifest.json:37]: @cristicismas the key value for title and text should not be "Blaze". The key value pair under params determines the query parameters and most of the applications use "title" for "title" and "text" for "text".

**Comment by cristicismas** [comment]
I've changed the 'blaze' value to 'title' and 'text'. Is it alright now, or do you need some specific values?

**Comment by abhishek-aa** [comment]
@cristicismas  It is alright now!

File: client/src/manifest.json (hunk 1 above)
Comment (blenderskool) [reply client/src/manifest.json]: Thanks for the review @abhishek-aa :smile:

**Comment by blenderskool** [comment]
@cristicismas I have tested this and it is working as expected. Thanks for the PR :tada:

**Pull request merged**